nymvpn-x/src-tauri/src/country.rs

```diff
@@ -0,0 +1,39 @@
+use std::fmt;
+
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+use ts_rs::TS;
+
+pub static FASTEST_NODE_LOCATION: Lazy<Country> = Lazy::new(|| Country {
+    code: String::from("DE"),
+    name: String::from("Germany"),
+});
+
+// TODO use countries requested from the backend instead of hardcoded ones
+pub static DEFAULT_ENTRY_COUNTRY: Lazy<Country> = Lazy::new(|| Country {
+    code: String::from("FR"),
+    name: String::from("France"),
+});
+pub static DEFAULT_EXIT_COUNTRY: Lazy<Country> = Lazy::new(|| Country {
+    code: String::from("CH"),
+    name: String::from("Switzerland"),
```

Comment:
This caused an error for me, with no Swiss exit gateways available

Reply:
yea good one, but as you can see given the `TODO` comment above, it will be covered in a later PR